content/whats-new/7.0-news.json

```
@@ -0,0 +1,44 @@
+[
+  {
+    "title":"Akeneo App Store",
+    "areas": ["Connectivity"],
+    "editions": ["EE", "GE"],
+    "since-version": "Serenity",
```

Can you change it for all lines ?
```suggestion
    "since-version": "7.0",
```